### PR TITLE
[java] Add null check before returning isDisplayed value

### LINKED ISSUE #9266: isDisplayed throws NullPointerException
## 🐛 Bug Report

Running with java binding and Firefox, the framework occasionally throws NullPointerException on the isDisplayed call.

**Please see https://github.com/mozilla/geckodriver/issues/1825 for more information.** As stated from geckodriver team, the issue is apparently related to the custom JS that is executed with this method. The ticket also contains trace logs. 

## To Reproduce

Hard to provide steps, as it happens "randomly".

## Expected behavior

Should not throw exception..

## Environment

OS: Windows 10
Browser: Firefox
Browser version: 84.0.1
Browser Driver version: geckodriver 0.28.0 (c00d2b6acd3f 2020-11-03 16:29 +0200)
Language Bindings version: java 3.141.59

### PR DESCRIPTION
**Pull request by pujagani**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Add null check before returning isDisplayed value.
Fixes #9266 from Java client end.

### Motivation and Context

Related to the issue mentioned above when isDisplayed() throws an NullPointerException randomly when running with the Firefox driver. To get the result of isDisplayed() function the script https://github.com/SeleniumHQ/selenium/blob/trunk/rb/lib/selenium/webdriver/atoms/isDisplayed.js is run on the driver end.  Sometimes the Firefox driver returns null and a null check would safeguard against that. It could be similar to https://github.com/SeleniumHQ/selenium/issues/4555.

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: java/client/src/org/openqa/selenium/remote/RemoteWebElement.java [hunk 1]
```diff
@@ -295,6 +295,9 @@ public boolean isDisplayed() {
     Object value = execute(DriverCommand.IS_ELEMENT_DISPLAYED(id))
       .getValue();
     try {
+      if (value == null) {
+        return false;
+      }
```
Comment (diemol) [inline java/client/src/org/openqa/selenium/remote/RemoteWebElement.java:301]:
```suggestion
      // See https://github.com/SeleniumHQ/selenium/issues/9266
      if (value == null) {
        return false;
      }
```

**Review by diemol — APPROVED** [review]
Thank you, @pujagani!

**Comment by PiotrZgadzaj** [comment]
I don't know what is selenium release calendar, but do you plan to include that in some selenium 3.x release? If so then when ?

**Comment by diemol** [comment]
We do not backport fixes to previous releases, so this will be available in `beta-3`, which should be released (I guess) by mid or end of April.

**Pull request merged**